Apps/Laboratory/test/unit/Controllers/LaboratoryControllerTests.cs

```diff
@@ -168,6 +178,10 @@ public async Task GetLabOrderError()
             Assert.True(result.ResultStatus == Common.Constants.ResultType.Error);
         }
 
+        /// <summary>
+        /// Test for GetLabReport.
```

Comment:
nit: this documentation does not add more than the function name

Reply:
just to pass the SA1600 as I don't think we need extra details for this :-)